### ISSUE TITLE
Identify PrivacyBadger in all log messages it generates

### ISSUE THREAD

**Issue by phanimahesh** [EFForg/privacybadgerfirefox#610]
Privacy badger writes a lot of stuff to console. Mix it together with a couple more extensions that do the same and now it is impossible to see what logged a particular item.

It is a general good practice to identify yourself in all the log messages. Can we do a bulk replace of `console.log(` with `console.log("PRIVACY-BADGER: " +` ?

**Comment by cooperq** [comment]
This seems like a reasonable request. I think a better way to architect it might be to implement our own logging subsystem so that logs can be off by default but enabled by a developer. It should also automatically prepend the string "PRIVACY_BADGER"

**Comment by phanimahesh** [comment]
Yep. I was going to suggest a logging subsystem but thought it would be overkill here.

The logging is not verbose currently, and it need not be turned off by default. A logging subsystem also provides the possibility to log at various levels and set logging level at runtime as desired. A user preference if created should be to control the logging level.

This is all pretty much standard logging functionality. There must be an existing implementation available.
